### PR TITLE
Flyout docs tweaks

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

File: src/app/components/flyout/index.html [hunk 1]
```diff
@@ -1,17 +1,12 @@
 <sky-demo-page
   pageTitle="Flyout"
   skyuxModule="SkyFlyoutModule"
-  packageName="@skyux/flyout"
 >
 
   <sky-demo-page-summary>
     <p>
-      The flyout service launches a container over a page's content to display large quantities of supplementary information related to a task. The <stache-code>SkyFlyoutService</stache-code> provider lauches flyouts and provides a common look-and-feel. The <stache-code>SkyFlyoutInstance</stache-code> class watches for and triggers flyout events.
+      The flyout service launches a container over a page's content to display large quantities of supplementary information related to a task. The <stache-code>SkyFlyoutService</stache-code> provider lauches flyouts and provides a common look and feel. This service dynamically generates the flyout component and appends it directly to the document's <stache-code>body</stache-code> element. The <stache-code>SkyFlyoutInstance</stache-code> class watches for and triggers flyout events.
```
Comment (Blackbaud-AlexKingman) [inline src/app/components/flyout/index.html]: I noticed a typo here - `lauches`. I think it was leftover from before?
Also, we are repetitive with that word in the first two sentences. Is there another way we could say this?

File: src/app/components/flyout/index.html [hunk 2]
```diff
@@ -39,52 +34,54 @@
...
     <sky-demo-page-property
       propertyName="ariaRole"
       isOptional="true"
     >
-      Sets the flyout's <stache-code>role</stache-code> attribute to support accessibility.
+      Specifies an ARIA role for the flyout to support accessibility by indicating how the flyout functions and what it controls. The ARIA role indicates what the flyout represents on the web page. For information about ARIA roles, see the <a href="https://www.w3.org/WAI/PF/aria/roles">WAI-ARIA roles model</a>. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="defaultWidth"
       defaultValue="500px"
```
Comment (Blackbaud-AlexKingman) [inline src/app/components/flyout/index.html]: Now that we're pointing out this is a `numeric` type, we should probably change `defaultValue="500px"` to `defaultValue="500"`

File: src/app/components/flyout/index.html [hunk 3]
```diff
@@ -39,52 +34,54 @@
...
       propertyName="maxWidth"
       defaultValue="defaultWidth"
       isOptional="true"
     >
-      Specifies the maximum resize width of the flyout container.
+      Specifies the maximum resize width of the flyout container. This property accepts <stache-code>number</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="minWidth"
       defaultValue="320px"
```
Comment (Blackbaud-AlexKingman) [inline src/app/components/flyout/index.html]: Same thing here - `320` instead of `320px`

File: src/app/components/flyout/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/components/flyout/index.html]: Good catches. Thanks. ... For the repetition, I'll tighten things up and combine the second and third sentences.

File: src/app/components/flyout/index.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/components/flyout/index.html]: Will do.

**Pull request merged**